tests/test_tags.py

```diff
@@ -1180,3 +1180,14 @@ def manylinux_compatible(tag_major, tag_minor, tag_arch):
             "linux_x86_64",
         ]
         assert platforms == expected
+
+    def test_pypy_first_tag(self, monkeypatch):
+        # When building the complete list of pypy tags, make sure the first
+        # <interpreter>-none-any one is pp3-none-any
+        monkeypatch.setattr(tags, "interpreter_name", lambda: "pp")
+
+        for tag in tags.sys_tags():
+            if tag.platform == "any":
```

Comment:
Just to be future-proof, could we also check that the ABI is `none`?

Reply:
I meant:
```suggestion
            if tag.abi == "none" and tag.platform == "any":
```

The question is whether it would ever make sense for this triple to come after anything that has the ABI set to something other than `none`. I think we agree it shouldn't, but to help guarantee that is the case, I was asking for the `break` trigger to be more explicit.